net-ftp/atftp: add 0.8.0

madmartin opened the pull request:
Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

Flowdalic commented on net-ftp/atftp/atftp-0.8.0.ebuild:
```diff
@@ -0,0 +1,61 @@
+	!net-ftp/tftp-hpa
+	!net-ftp/uftpd
+	selinux? ( sec-policy/selinux-tftp )"
+BDEPEND="${DEPEND}"
```
Are you sure that all of DEPEND are also BDEPEND?

madmartin replied:
@Flowdalic IMHO all three of 

```
DEPEND="tcpd? ( sys-apps/tcp-wrappers )
    readline? ( sys-libs/readline:0= )
    pcre? ( dev-libs/libpcre2 )"
```

are needed at build time, yes.
Do you have a different opinion?

Flowdalic replied:
Yes they are build dependencies, but I suspect they are not BDEPEND dependencies, just DEPEND dependencies.

Flowdalic commented on net-ftp/atftp/atftp-0.8.0.ebuild:
```diff
@@ -0,0 +1,61 @@
+BDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-CFLAGS.patch"
```
That file is identical the to the already existing `atftp-0.7.5-CFLAGS.patch`. Simply continue using it.
```suggestion
	"${FILESDIR}/${PN}-0.7.5-CFLAGS.patch"
```

Flowdalic commented on net-ftp/atftp/atftp-0.8.0.ebuild:
```diff
@@ -0,0 +1,61 @@
+
+DEPEND="tcpd? ( sys-apps/tcp-wrappers )
+	readline? ( sys-libs/readline:0= )
+	pcre? ( dev-libs/libpcre2 )"
```
```suggestion
	pcre? ( dev-libs/libpcre2:= )"
```

madmartin replied:
maybe a good idea....

Flowdalic reviewed (changes requested):
Tests fail for me. Any reason why you dropped `src_test` from the previous version's ebuild? If I re-add it, then the tests run.

Flowdalic commented:
@madmartin friendly ping

madmartin commented:
> Tests fail for me. Any reason why you dropped `src_test` from the previous version's ebuild? 

You wrote it in your first sentence. I discovered test.sh fails when used in portage build system/sandbox.

It runs well when started after merge. I did not want to loose time for investigation, so I dropped `src_test()` for now from the ebuild.

> grep: ./atftp-test.CD1J8l/multicast*.log: No such file or directory

is not looking nice but harmless. no logs from multicast test found. I will fix this.

> ./test.sh: line 580: 5 + /data-scratch/var-tmp/portage/net-ftp/atftp-0.8.0/image : syntax error: operand expected (error token is "/data-scratch/var-tmp/portage/net-ftp/atftp-0.8.0/image ")

is more difficult to fix. My first diagnose is that $0 behaves different between regular user shell and portage `src_test()`.

That's why I removed `src_test()` from the ebuild.
